indico/types/submission.py

```diff
@@ -71,32 +77,53 @@ class Submission(BaseType):
     to generate the result file.
 
     Attributes:
-        id (int): the Submission id
-        dataset_id (int): the Dataset id
-        workflow_id (int): the Workflow id
+        id (int): The Submission id
+        dataset_id (int): The Dataset id
+        workflow_id (int): The Workflow id
         status (str): status of the submission. One of
             {VALID_SUBMISSION_STATUSES}
-        input_files (list[SubmissionFile]): the SubmissionFiles for the Submission
+        created_at (datetime): Datetime the submission was created
+        updated_at (datetime): Datetime the submission was updated
+        created_by (int): Id of the user who created the submission
+        updated_by (int): Id of the user who updated the submission
+        completed_at (datetime): Datetime the submission reached a completed state
+        files_deleted (bool): Submission files have been deleted (True) or not deleted (False) from file store
+        input_files (List[SubmissionFile]): The SubmissionFiles for the Submission
         input_file (str): URL of the first input datafile within the Indico Platform.
-        input_filename (str): name of the first original file
+        input_filename (str): Name of the first original file
         result_file (str): URL of the result datafile within the Indico Platform
```

Comment:
maybe time to update the docs here as well. `URL of the _latest_ result file for this submission` or something to that effect?